hazelcast/src/main/java/com/hazelcast/map/impl/operation/DefaultMapOperationProvider.java

```diff
@@ -171,8 +171,8 @@ public MapOperation createTxnDeleteOperation(String name, Data dataKey, long ver
 
     @Override
     public MapOperation createTxnLockAndGetOperation(String name, Data dataKey, long timeout, long ttl, String
-            ownerUuid) {
-        return new TxnLockAndGetOperation(name, dataKey, timeout, ttl, ownerUuid);
+            ownerUuid, boolean loadKey) {
```

Comment:
Afaik the whole entry is loaded; key & value. Perhaps it makes more sense to call it loadFromStorage?